cmd/bench/bench.go

```diff
@@ -36,7 +36,7 @@ func main() {
 		profFilePath := path.Join(dir, "./CPU.prof")
 		fmt.Printf("CPU profile: %s\n", profFilePath)
 
-		f, err := os.Create(profFilePath)
+		f, err := os.Create(profFilePath) //#nosec G304
```

Comment:
this can just be

```suggestion
		f, err := os.Create("./CPU.prof")
```

Reply:
Great, thanks! I slightly modified it for logging purposes.